tools/kotlin-native-gradle-plugin/src/test/groovy/org/jetbrains/kotlin/gradle/plugin/test/CMakeSpecification.groovy

```diff
@@ -1,17 +1,20 @@
 package org.jetbrains.kotlin.gradle.plugin.test
 
-class CMakeSpecification extends BaseKonanSpecification {
+import org.jetbrains.kotlin.konan.target.Family
+import org.jetbrains.kotlin.konan.target.TargetManager
 
+class CMakeSpecification extends BaseKonanSpecification {
     def 'Plugin should generate CMake from project without additional settings'() {
         when:
+        def sdlIncludePath = (TargetManager.host.family == Family.WINDOWS) ? "C:/SDL2/include" : "/usr/include/SDL2"
```

Comment:
Hm, I think we can make this test more interesting if, insider of string interpolation in `build.gradle`, we write

```
interop('sdl') {
    target('mingw') { includeDirs "C:\\SDL2\\include" }
    target('linux') { includeDirs "/usr/include/SDL" }
}
```

and then, for CMake, use string interpolation as is already done.

Reply:
Good idea, I'll try it.